scrunch/datasets.py

```diff
@@ -1882,3 +1989,56 @@ def move(self, path, position=-1):
             )
         target_group = self.dataset.order[str(path)]
         target_group.insert(self.alias, position=position)
+
+    def summary(self, names=False, tablefmt='psql'):
+        dimensions = [{'variable': self.alias}]
+        cube = self.dataset.get_cube(dimensions=dimensions)
+        cube_result = cube['value']['result']
+        counts = cube_result['counts']
+        param = 'name' if names else 'alias'
+        valid_formats = 'plain simple grid fancy_grid pipe orgtbl jira psql ' \
+                        'rst mediawiki moinmoin html latex latex_booktabs ' \
+                        'textile'.split(' ')
+        if tablefmt not in valid_formats:
+            raise ValueError(
+                'format must be one of: %s' % ', '.join(valid_formats))
+
+        subvars = self.shoji_tuple.get('subvariables')
+        if subvars:
+            refs = cube_result['dimensions'][1]['references']
+            subrefs = refs['subreferences']
+            categories = cube_result['dimensions'][1]['type']['categories']
+
+            table = []
+            i = 0
+            for ref in subrefs:
+                row = [ref[param]]
+                for c in categories:
+                    row.append(counts[i])
+                    i += 1
+                table.append(row)
+            headers = [param] + [c['id'] for c in categories]
+
+        else:
+            refs = cube_result['dimensions'][0]['references']
+            categories = cube_result['dimensions'][0]['type']['categories']
+
+            table = []
+            for num, c in enumerate(categories):
+                table.append([c['name'], counts[num]])
+            headers = ['category', 'count']
+
+        print('\n      alias: {alias}, id: {id}\n'
```

Comment:
You shouldn't print to stdout by default. Make the function receive a StringIO and write to that. Then you can test the output or you can export this to a file if desired.